decidim-proposals/spec/shared/manage_proposals_examples.rb

```diff
@@ -198,6 +198,35 @@
             expect(page).to have_selector("img[src*=\"city.jpeg\"]", count: 1)
           end
         end
+
+        context "when proposals comes from a meeting" do
+          let!(:meeting_component) { create(:meeting_component, participatory_space: participatory_process) }
+          let!(:meetings) { create_list(:meeting, 3, component: meeting_component) }
+
+          it "creates a new proposal with meeting as author" do
+            click_link "New proposal"
+
+            within ".new_proposal" do
+              fill_in :proposal_title, with: "Proposal with meeting as author"
+              fill_in :proposal_body, with: "Proposal body of meeting as author"
+              execute_script("$('#proposal_created_in_meeting').change()")
+              find(:css, "#proposal_created_in_meeting").set(true)
+              select translated(meetings.first.title), from: :proposal_meeting_id
+              select category.name["en"], from: :proposal_category_id
+              find("*[type=submit]").click
+            end
+
+            expect(page).to have_admin_callout("successfully")
+
+            within "table" do
+              proposal = Decidim::Proposals::Proposal.last
+
+              expect(page).to have_content("Proposal with meeting as author")
+              expect(proposal.body).to eq("Proposal body of meeting as author")
+              expect(proposal.category).to eq(category)
```

Comment:
Is there an easy way to check from the UI the author?

Reply:
in admin site, not! We can add a new field, showing the author when is official or meeting official. What you think?

Reply:
I've been checking now, and there are a lot of information in admin proposals views. If we add another field, it could be caotic
Check the screenshot
![screenshot-localhost-3000-2018 11 09-09-10-39](https://user-images.githubusercontent.com/28536082/48250689-70986000-e3ff-11e8-991e-f54d5d700e52.png)

Reply:
Let's leave it like this then